### ISSUE TITLE
Wrathgate and its aftermath Alliance pov – missing cinematic and delayed roleplay

### ISSUE THREAD

**Issue by Greywolf59** [gamefreedomgit/Frostmourne#465]
[Return to Angrathar](https://www.wowhead.com/wotlk/quest=12499/return-to-angrathar)
- Handing in the quest should prompt cutscene to play. This does not happen.
- [Alexstrazsa](https://www.wowhead.com/wotlk/npc=31333/alexstrasza-the-life-binder) has gossip option to replay the cutscene, however clicking it does not do anything.
- https://wowpedia.fandom.com/wiki/Return_To_Angrathar_(Alliance)?so=search
- https://www.youtube.com/watch?v=Ch4rc5W4dKY&ab_channel=WorldofWarcraft

[Fate, Up Against Your Will](https://www.wowhead.com/wotlk/quest=13369/fate-up-against-your-will)
- When you teleport to Thrall, there should be a bit of roleplay happening. This roleplay either: does not happen at all, is delayed, or is prompted by something else.
- Situation: Whole my time in Orgrimmar nothing happened. I have handed in the quest, accepted the new one and waited a bit. Then I decided to go to the teleporter back to Stormwind. As I was clicking the portal, I have seen visual effect of other NPC teleporting in but it was too late, the click was made. I have spent about 2 or 3 minutes in Orgrimmar.
- The scenario should happen right away as you teleport to Orgrimmar.
- https://wowpedia.fandom.com/wiki/Fate,_Up_Against_Your_Will#Dialogue
- https://www.wowhead.com/wotlk/quest=13369/fate-up-against-your-will#comments:id=460036
- https://www.youtube.com/watch?v=jXJFbNVk8BE&ab_channel=FamouzPlayz

**Comment by UltraNix** [comment]
```
[Return to Angrathar](https://www.wowhead.com/wotlk/quest=12499/return-to-angrathar)

Handing in the quest should prompt cutscene to play. This does not happen.
[Alexstrazsa](https://www.wowhead.com/wotlk/npc=31333/alexstrasza-the-life-binder) has gossip option to replay the cutscene, however clicking it does not do anything.
https://wowpedia.fandom.com/wiki/Return_To_Angrathar_(Alliance)?so=search
https://www.youtube.com/watch?v=Ch4rc5W4dKY&ab_channel=WorldofWarcraft
```
Just tested and works okay. Maybe you are missing this movie in your files?

**Comment by Greywolf59** [comment]
I have the fresh client from your site, always running in admin mode. I can check later on if it is there though. Do you know the name od the file?

**Comment by UltraNix** [comment]
Path: .../Data/<your_local>/Interface/Cinematics
File: WOW_INTRO_LK

**Comment by Greywolf59** [comment]
Yup, it is actually missing in my files for some reason.

**Comment by UltraNix** [comment]
Fixed.

**Comment by Zeehra** [comment]
This does still not work. 
I have the full client, the newest patch, and the cinematic file is missing. I found no link in this thread to a download of said file. But even without the cinematic, there should still be a follow up quest and a phasing. There is some phasing going on if you ride into the court of skulls from the south, but there is no sight of Alexstrasza - and I don't fancy finding out where the borders of this particular phasing is on a hc character, just to find myself in the midst of 60 lvl80 elites, trying to locate a dragon that isn't there. 
How do I download the cutscene file, and/or how do I continue this questline? I made a ticket and a GM just told me to comment here.
